src/main/java/org/terasology/quests/examples/QuestPointEntityProvider.java

```diff
@@ -32,6 +32,7 @@
 import org.terasology.world.generation.Region;
 import org.terasology.world.generation.facets.SurfaceHeightFacet;
 import org.terasology.world.generator.plugin.RegisterPlugin;
+import org.terasology.books.logic.BookComponent;
```

Comment:
Even though I have added Books as dependency, this is flagged as error in my IntelliJ IDE, and I am getting error "could not resolve :module:Books:" error while compiling.

Reply:
Resolved by running `gradlew idea`